### ISSUE TITLE
Failure on Travis (PHP project) with ESLint v3.0.1 (strip-bom)

### ISSUE THREAD

**Issue by flodolo** [eslint/eslint#6607]
We use ESLint in Travis on a PHP project to lint some .js files, and started having failure with v3.0.1

```
/home/travis/.nvm/v0.10.36/lib/node_modules/eslint/node_modules/strip-bom/index.js:2

module.exports = x => {
                    ^
SyntaxError: Unexpected token >
    at Module._compile (module.js:439:25)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/home/travis/.nvm/v0.10.36/lib/node_modules/eslint/lib/config/config-file.js:23:16)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
```

That piece of code comes from strip-bom, and I noticed a change in dependency
https://github.com/eslint/eslint/compare/v3.0.0...v3.0.1#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R67

**Comment by flodolo** [comment]
In case someone else has the issue, fixed by updating node in Travis (v0.10.36 is the one installed by default).
```
before_install:
  ## Install latest version of node
  - nvm install node
```

**Comment by The-Compiler** [comment]
@flodolo Thanks for mentioning this - this fixed it for me as well: https://github.com/The-Compiler/qutebrowser/commit/89cdef851d8f56b509dc04331330dd14f8ec62a1
